content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -200,6 +200,20 @@ If the access URLs of the restored cluster is changed from the previous cluster,
 
 If the majority of etcd members have permanently failed, the etcd cluster is considered failed. In this scenario, Kubernetes cannot make any changes to its current state. Although the scheduled pods might continue to run, no new pods can be scheduled. In such cases, recover the etcd cluster and potentially reconfigure Kubernetes API server to fix the issue.
 
+{{< note >}}
+In general, we don't recommend restoring etcd when any apiserver is running. We
+recommend:
+
+- stop *all* kube-apiserver instances
+- restore state in all etcd instances
+- restart all kube-apiserver instances
+
+We also recommend restarting any components to ensure that they don't
+rely on some stale data. Note that in practice, given that the restore takes a
+bit of time, the critical components will loose leader lock and they will
+restart themselves.
```

Comment:
What do you think about:
During the restoration, critical components lose leader lock and restart themselves. OR
While the restoration occurs, critical components lose leader lock and restart themselves.

Reply:
updated. Please take a look